tests/reducers/globalPropertyEncounterTypeReducer.test.js

```diff
@@ -0,0 +1,58 @@
+import deepFreeze from 'deep-freeze';
+import * as actions from '../../app/js/actions/globalPropertyEncounterType';
```

Comment:
Is it possible to import only the required modules using their names instead of importing all modules?